cobalt/browser/application.cc

```diff
@@ -732,6 +732,11 @@ Application::Application(const base::Closure& quit_closure, bool should_preload,
     options.renderer_module_options.enable_fps_overlay = true;
   }
 
+  if (command_line->HasSwitch(browser::switches::kRasterizerType)) {
```

Comment:
This seems fine for Cobalt, but what about our other test binaries that depend on rasterizer ? E.g. renderer_test, sandboxes, layout_tests, WPT and others ?

Reply:
As discussed offline, this is fine for all the tests. We only override this variable when there is a command line switch, and we leave it as default (what we have today for each platform for all those tests.)

Reply:
As per our conversation, no need to solve those right away. We can just add the switch to Cobalt and deal with CI / Starboard config consolidation in later stage.